components/prism-rescript.js

```diff
@@ -0,0 +1,57 @@
+(function (Prism) {
+	var operator = /\.{3}|:[:=]|\|>|->|=(?:==?|>)?|<=?|>=?|[|^?'#!~`]|[+\-*\/]\.?|\b(?:asr|land|lor|lsl|lsr|lxor|mod)\b/;
+	var className = /(\b[A-Z]\w*)|(@[a-z.]*)|#[A-Za-z]\w*|#\d/;
+	var keyword = /\b(?:and|as|assert|begin|bool|class|constraint|do|done|downto|else|end|exception|external|float|for|fun|function|if|in|include|inherit|initializer|int|lazy|let|method|module|mutable|new|nonrec|object|of|open|or|private|rec|string|switch|then|to|try|type|val|virtual|when|while|with)\b/;
+
+	Prism.languages.rescript = {
+		'comment': {
+			pattern: /\/\/(?:[^\r\n\\]|\\(?:\r\n?|\n|(?![\r\n])))*|\/\*[\s\S]*?(?:\*\/|$)/,
+			greedy: true
+		},
+		'string': /"(?:\\(?:\r\n|[\s\S])|[^\\\r\n"])*"/,
```

Comment:
This still needs to be greedy. Otherwise, `"https://example.com"` will be highlighted incorrectly.

Reply:
Done